src/components/admin/personen/PasswordReset.vue

```diff
@@ -44,6 +44,56 @@
     isActive.value = false;
     emit('onClearPassword');
   }
+
+  // Used to enable <style> in printPassword, otherwise blocked by CSP
+  declare var cspNonce: string;
```

Comment:
Global declarations wie sowas muss in einer separaten Datei gemacht werden. global.d.ts erstellen neben types.d.ts und bitte da machen. TS hat direkten Zugriff auf solche Variablen und die müssen nicht extra mit "export" deklariert werden.